### ISSUE TITLE
Keras2 LSTM layer: How right now define U_regularizer which was in Keras1

### ISSUE THREAD

**Issue by vladimircape** [keras-team/keras#10822]
I want to write LSTM layer using Keras2 ,i found all changes of migration to second version 
except for U_regularizer, how to define it?

> LSTM(rnn_size, return_sequences=True, # batch_norm=batch_norm,
>                 kernel_regularizer=regularizer, U_regularizer=regularizer,
>                 bias_regularizer=regularizer, dropout=p_W, recurrent_dropout=p_U,
>                 name='lstmvova_%d'%(i+1)

I think it's recurrent_regularizer
Am i right?
